library/cluster_vm.py

```diff
@@ -67,10 +67,14 @@
         arguments I(name), I(metadata_name)
       - C(define_colocation) Define colocation rules. Require
         arguments I(name), I(colocated_vms)
+      - C(add_pacemaker_remote) Add a pacemaker remote. Require arguments
+        I(name), −I(remote_name), I(remote_address)
+      - -C(remove_pacemaker_remote) Remove a pacemaker remote. Require arguments
+         I(name)
```

Comment:
Remove extra `-` on line 72, and extra indentation in line 73

Reply:
Done